src/VisualStudio/Core/Def/Implementation/LanguageClient/RazorInProcLanguageClient.cs

```diff
@@ -38,7 +38,7 @@ internal class RazorInProcLanguageClient : AbstractInProcLanguageClient
         /// <summary>
         /// Gets the name of the language client (displayed to the user).
```

Comment:
Should I remove the "displayed to the user" portion? Given that it's only in yellow bars 😆

Reply:
Maybe we can specify that it's used just for yellow bar cases?